### PR TITLE
Adding Forcepoint NGFW Advanced Workbook

### PR DESCRIPTION
**Pull request by RaGerges**
Change(s):
   -New Workbook

   Reason for Change(s):
   - This workbook provides a complete solution for advanced security and application insights, by correlation of Forcepoint firewall logs with Microsoft threat intelligence indicators, as well as monitoring Forcepoint loggers health and performance.

   Version Updated:
   - This is version 1.0

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

**Comment by RaGerges** [comment]
Closed for missing to fill the required details

**Comment by RaGerges** [comment]
Comments updated successfully.

File: Workbooks/Images/Logos/FPAdvLogo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...618 179.271 313.052 179.271 L 307.486 179.271 307.486 200.000 M191.768 188.627 C 193.837 189.591,196.251 192.077,196.886 193.901 L 197.273 195.010 187.312 195.010 C 181.834 195.010,177.351 194.916,177.351 194.802 C 177.351 190.005,186.402 186.129,191.768 188.627 M74.021 189.279 C 81.484 193.008,82.745 202.747,76.505 208.450 C 68.989 215.320,57.404 209.410,58.060 199.040 C 58.587 190.701,66.807 185.674,74.021 189.279 M238.627 189.186 C 248.905 194.489,245.668 211.116,234.357 211.116 C 224.449 211.116,220.306 197.290,228.191 190.540 C 231.058 188.086,235.402 187.523,238.627 189.186 M286.499 189.250 C 297.149 194.628,293.435 211.359,281.601 211.314 C 270.968 211.273,266.279 197.910,274.392 190.769 C 277.614 187.932,282.641 187.302,286.499 189.250 M13.820 210.173 L 13.820 220.729 19.770 220.729 L 25.720 220.729 25.720 210.173 L 25.720 199.616 19.770 199.616 L 13.820 199.616 13.820 210.173 " stroke="none" fill="#1c242c" fill-rule="evenodd"></path><path id="path2" d="" stroke="none" fill="#108ca0" fill-rule="evenodd"></path><path id="path3" d="" stroke="none" fill="#104468" fill-rule="evenodd"></path><path id="path4" d="" stroke="none" fill="#10482c" fill-rule="evenodd"></path></g></svg>
```
Comment (v-sabiraj) [inline Workbooks/Images/Logos/FPAdvLogo.svg:1]: Hey @RaGerges, please use GUID as id, please check and update, thanks.

Comment (v-sabiraj) [inline Workbooks/Images/Logos/FPAdvLogo.svg:1]: Hey @RaGerges, Do not use illegal attributes like data-name, style, **xmlns:link**, please remove  xmlns:link as it is an illegal attribute, thanks.

**Review by v-sabiraj — CHANGES_REQUESTED** [review]

File: Workbooks/Images/Logos/FPAdvLogo.svg (hunk 1 above)
Comment (RaGerges) [reply Workbooks/Images/Logos/FPAdvLogo.svg]: Dear @v-sabiraj. GUIDs used for id attributes. Thanks

Comment (RaGerges) [reply Workbooks/Images/Logos/FPAdvLogo.svg]: Dear @v-sabiraj . xmlns:xlink removed. Thanks

File: Workbooks/ForcepointNGFWAdvanced.json [hunk 2]
```diff
@@ -0,0 +1,1472 @@
...
+        "value": "LH"
+      },
+      "name": "group - 6"
+    }
+  ],
+  "fallbackResourceIds": [
+    "/subscriptions/84b4e155-0714-409e-9c04-10b3d71534ff/resourcegroups/rg-eng-internal-siem-ext/providers/microsoft.operationalinsights/workspaces/loga-eng-internal-siem-ext"
+  ],
+  "styleSettings": {},
+  "fromTemplateId": "sentinel-UserWorkbook",
```
[the remark this answers is not in the document]
Comment (RaGerges) [reply Workbooks/ForcepointNGFWAdvanced.json]: Dear @v-sabiraj . Attribute updated. Thanks

**Comment by RaGerges** [comment]
Dear @v-sabiraj, all comments have been taken care of. Thanks

**Comment by RaGerges** [comment]
Dear @v-sabiraj . All validations passed.

**Review by vmanojreddy — APPROVED** [review]

**Pull request merged**